docs/spatial-vector.rst

```diff
@@ -4,6 +4,45 @@ Earthpy Spatial Vector Data
 The ``earthpy`` spatial module provides functions that wrap around the ``rasterio``
 and ``geopandas`` to work with raster and vector data in Python.
 
+Earthpy Spatial Vector Data
+===========================
+
+The ``earthpy`` spatial vector module provides functions that wrap around ``geopandas`` to work with vector data in Python.
+
+Both layers must be the same Coordinate Reference System (CRS).
+
+Clip Points
+-----------
+
+The ``clip_point`` function takes a shapefile to clip to another shapefile. 
```

Comment:
Update line 17 to: `The ``clip_point`` function clips vector points (Geopandas dataframe) to a polygon vector object (Geopandas dataframe).  It can be executed independently or using the ``clip_shp`` function.`